woocommerce/woocommerce-gateway-stripe
# Add general section UI

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1760

# Description: 
Add a section to the "Settings" tab on the main Stripe gateway. This is for UI purpose only, modal is not included.

# Testing instructions
1. Go to `wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe`
2. Click "Settings" tab
3. Verify the the settings card matches the design, like the following:
![image](https://user-images.githubusercontent.com/572862/132579186-c6e9a451-1a47-4079-8b35-668da203f1f1.png)
